project/assets/scripts/controller/new-evaluation/new-evaluation.js

```diff
@@ -1,68 +1,78 @@
-        removeEvents('submit-button', getFormData);
-    };
-};
 
-function getSelected(selectId) {
-    const e = document.getElementById(`${selectId}`);
-    return e.options[e.selectedIndex].value;
-}
+    function getSelected(selectId) {
+        const e = document.getElementById(`${selectId}`);
+        return e.options[e.selectedIndex].value;
+    }
 
-function getTextarea() {
-    const textareaArray = getTextareaArray().items.map((el) => {
-        const textareaKey = constructId(el.title);
-        const textareaObj = {};
-        textareaObj[textareaKey] = document.getElementById(constructId(el.title)).value;
-        return textareaObj;
-    });
-    return textareaArray;
-}
+    function getTextarea() {
+        const textareaArray = feedbackApp.data.newEvaluation.textareaItems().items.map((el) => {
+            const textareaKey = constructId(el.title);
+            const textareaObj = {};
+            textareaObj[textareaKey] = document.getElementById(constructId(el.title)).value;
+            return textareaObj;
+        });
+        return textareaArray;
+    }
 
-function getFieldset() {
-    const fieldsetArr = getTechnicalAreaArray().map((el) => {
-        const obj = {};
-        const objKey = constructId(el.legend);
-        obj[objKey] = {};
-        const dropDownObj = {};
-        el.dropDownOptions.forEach((op) => {
-            op.forEach((i) => {
-                const dropDownKey = constructIdFieldset(el.legend, i.label);
-                dropDownObj[dropDownKey] = getSelected(constructIdFieldset(el.legend, i.label));
-                obj[objKey] = dropDownObj;
+    function getFieldset(data) {
+        const fieldsetArr = data.map((el) => {
+            const obj = {};
+            const objKey = constructId(el.legend);
+            obj[objKey] = {};
+            const dropDownObj = {};
+            el.dropDownOptions.forEach((op) => {
+                op.forEach((i) => {
+                    const dropDownKey = constructIdFieldset(el.legend, i.label);
+                    dropDownObj[dropDownKey] = getSelected(constructIdFieldset(el.legend, i.label));
+                    obj[objKey] = dropDownObj;
+                });
             });
+            return obj;
         });
-        return obj;
-    });
-    return fieldsetArr;
-}
+        return fieldsetArr;
+    }
 
-function getFormData() {
-    inputData = {
-        candidate: document.getElementById('candidate').value,
-        interviewer: document.getElementById('interviewer').value,
-        date: document.getElementById('date').value,
-    };
 
-    technicalLevel = document.querySelector('input[name="level"]:checked').value;
+    function prepareForm(data) {
+        inputData = {
+            candidate: document.getElementById('candidate').value,
+            interviewer: document.getElementById('interviewer').value,
+            date: document.getElementById('date').value,
```

Comment:
Stray comma at EOL. Not a huge issue, but I do like to complain about the details! :P

Reply:
Blame ESLint :))

Reply:
How dare you, ESLint?!? Shame, shame, shame!